lasagne/objectives.py

```diff
@@ -145,7 +145,9 @@ def binary_crossentropy(predictions, targets):
 
 
 def categorical_crossentropy(predictions, targets):
-    """Computes the categorical cross-entropy between predictions and targets.
+    """Computes the categorical cross-entropy between predictions $p_{i,j}$
+        and targets $t_{i,j}$, where $i$ denotes the data point and $j$
+        denotes the class.
```

Comment:
Sounds good, but you completely messed up the formatting :)

a) the first sentence must be short since it's used for the overview table
b) the text must not be indented otherwise it's assumed to be a code block
c) the math syntax in Sphinx does not use $

Please always compile and look at the documentation when changing docstrings in a nontrivial way: http://lasagne.readthedocs.io/en/latest/user/development.html#documentation

So the explanation of p, t, i, j should come after the formula, or in a second sentence before the formula, not in the first sentence.